docs/extensions/scala.md

```diff
@@ -54,6 +54,13 @@ export interface ScalaTestParams {
   /** The test classes to be run in this test execution.
    * It is the result of `buildTarget/scalaTestClasses`. */
   testClasses?: ScalaTestClassesItem[];
+
+  /** The JVM options to run tests with. In case JVM options 
```

Comment:
I don't think this is what we want because servers cannot find conflicts easily between jvm options. That operation requires parsing and sometimes even a semantic understanding of the JVM options, as some of them use keywords such as "Enable" or "Disable" which cancel each other.
  
I think the right semantics for this new addition should be that they override the JVM test options specified in the build.

Reply:
isn't the semantics of concatenation already that options later in the list override earlier ones?

Reply:
Concatenation doesn't solve the problem because there are JVM options that can't be enabled when other JVM options are and viceversa, so when the JVM finds that it will fail early and exit.

That's why only the JVM options passed by the user have to be used uniquely by the build server, there's no sensible merge strategy, that needs to be done by the user changing the JVM options.

Reply:
Yup, you're right. Good catch.

Reply:
OK, correct me if I'm wrong. The decision here is to state that jvm options defined by user will be overwritten by jvm options defined by build server is they collide, right?

Reply:
No, the solution is that the semantics of this field needs to change completely. All and only the JVM options provided here will be used by the build server to test the application, there's no merge strategy with the ones the build server has.

Reply:
OK, thanks for clarification.